data/test_stories/story_slot_different_types.md

```diff
@@ -0,0 +1,6 @@
+## story: Customer loses a credit card, reviews transactions, and gets a new card
```

Comment:
optional: that's somewhat more helpful as the current title (the file name already suggests it so consider it optional). Same applies to the yaml version of this
```suggestion
## story: Story with all different slot types
```